src/cmake/dependencies.cmake

```diff
@@ -117,7 +117,7 @@ if(NOT USE_SYSTEM_MAGNUM)
 
   # These are enabled by default but we don't need them right now -- disabling
   # for slightly faster builds. If you need any of these, simply delete a line.
-  set(WITH_DEBUGTOOLS OFF CACHE BOOL "" FORCE)
+  # set(WITH_DEBUGTOOLS OFF CACHE BOOL "" FORCE)
```

Comment:
I'd say just remove this line, DebugTools might be soon used for shader tests as well :)